pkg/asset/templates/kube-scheduler.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
     k8s-app: kube-scheduler
     version: v1.3.6_coreos.0
 spec:
+  selector:
+    matchLabels:
+      k8s-app: kube-scheduler
+  replicas: 2
```

Comment:
It keeps the update code more generic. I could add some logic where if replicas == 1, first scale up 2 to, update, and then scale back down.